modules/social.py

```diff
@@ -136,6 +138,10 @@ async def _generate_profile_card(self, member: discord.Member) -> discord.File:
         if dbUser['friendcode']:
             draw.text((350, 330), dbUser['friendcode'], (87, 111, 251), font=subtextFont)
 
+        if dbUser['smmcode']:
+            draw.text((350, 385) if dbUser['friendcode'] else (350, 330),
```

Comment:
This doesn't have to be split lines, plus I'm not sure if this would look well. Prehaps attach screenshot with example.